instances/latest/terminologies/disease/coronavirusDisease.jsonld

```diff
@@ -0,0 +1,21 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/disease/coronavirusDisease",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Disease",
+  "definition": "A Coronavirus infectious disease that is characterized by fever, cough and shortness of breath and that has_material_basis_in SARS-CoV-2.",
```

Comment:
Rather:
"An infectious disease caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), and typically presents with fever, cough and shortness of breath."